src/node-srt.cc

```diff
@@ -389,7 +389,7 @@ Napi::Value NodeSRT::EpollUWait(const Napi::CallbackInfo& info) {
   Napi::Number epidValue = info[0].As<Napi::Number>();
   Napi::Number msTimeOut = info[1].As<Napi::Number>();
 
-  int fdsSetSize = 100;
```

Comment:
this might as well be a `#define FD_SET_SIZE 100` while we are at it. No need to make it more complicated than that for any compiler :)

Reply:
also, ideally this value could be an optional parameter of the method, and the value here is only the default.

Reply:
Yeah, this was a fast thing to be able to compile it asap and do my things. Actually the `#define FD_SET_SIZE 100` seems pretty good if you are going to keep it static.